spec/index.bs

```diff
@@ -329,6 +350,7 @@ interface GPUAdapter {
     readonly attribute DOMString name;
     readonly attribute sequence<GPUExtensionName> extensions;
     //readonly attribute GPULimits limits; Don't expose higher limits for now.
+    readonly attribute boolean hasMajorPerformanceCaveat;
```

Comment:
How about having the powerPreference too?

Reply:
I think I want to avoid letting applications hard-filter on powerPreference, but it's a possibility.